README.md

```diff
@@ -27,7 +27,7 @@ Starting with macOS Sierra, Apple has begun enforcing the way in which you can s
 
 When you start deploying this script to your end-users you will want to ensure that it is scoped properly. At that very least, you'll want to create a Smart Group to determine if the target system(s) meet the system requirements for the macOS upgrade.
 
-* [laurentpertois/High-Sierra-Compatibility-Checker](https://github.com/laurentpertois/High-Sierra-Compatibility-Checker)
```

Comment:
This project supports to upgrade to High Sierra yet. So I'd like to keep this line.

Reply:
macOSUpgrade works perfectly well for Mojave too. Might be worth adding both High Sierra and Mojave checker links then? I had started to hack up the High-Sierra checker for Mojave before I realised there is a perfectly good one for Mojave.

Reply:
Yes. I hope that there are both statements.
Please coexist with the description for Mojave.